convert.go

```diff
@@ -199,7 +199,11 @@ func findFiles(dir, prefix string, recursive bool, toc *[]Asset, ignore []*regex
 		if strings.HasPrefix(asset.Name, prefix) {
 			asset.Name = asset.Name[len(prefix):]
 		} else {
-			asset.Name = filepath.Join(dir, file.Name())
+			if fi.IsDir() {
+				asset.Name = filepath.Join(dir, file.Name())
```

Comment:
I'm confused about this one. If it's a directory, we _don't_ want to create an asset for it, right?

Reply:
if you're searching a directory like "testdata/" and you have a file like "asset.png", then you want to join them to make "testdata/asset.png". But if you specified the full file name on the command line like "testadata/asset.png" then you dont want to append the file name again and end up with "testdata/asset.png/asset.png".

This bug only crops up when specifying a prefix that doesnt match the file youre looking at, otherwise it will just set it to the asset.Name

Reply:
Yeah you're right. FYI this is `dirOrFile` now and `file` has been changed to `entry`